python/pyspark/sql/types.py

```diff
@@ -1619,11 +1619,47 @@ def to_arrow_type(dt):
         arrow_type = pa.decimal(dt.precision, dt.scale)
     elif type(dt) == StringType:
         arrow_type = pa.string()
+    elif type(dt) == DateType:
+        arrow_type = pa.date32()
+    elif type(dt) == TimestampType:
+        arrow_type = pa.timestamp('us', tz='UTC')
```

Comment:
I think it is ok to specify 'UTC' here.  In Spark the `ArrowColumnVector` requires a timezone to be set because it expects the Arrow `NullableTimeStampMicroTZVector` but it doesn't do anything with the tz, so as long as the data is UTC time internally, it should be fine for Spark to use.

Reply:
I think It'd be nicer if we have some comments for this explanation.